crates/wasmparser/src/parser.rs

```diff
@@ -538,27 +527,13 @@ impl Parser {
                 }
 
                 // Check for custom sections (supported by all encodings)
-                if id == 0 {
-                    let start = reader.original_position();
-                    let range = Range {
-                        start,
-                        end: reader.original_position() + len as usize,
-                    };
-                    let mut content = subreader(reader, len)?;
-                    // Note that if this fails we can't read any more bytes,
-                    // so clear the "we'd succeed if we got this many more
-                    // bytes" because we can't recover from "eof" at this point.
-                    let name = content.read_string().map_err(clear_hint)?;
-                    return Ok(Payload::CustomSection {
-                        name,
-                        data_offset: content.original_position(),
-                        data: content.remaining_buffer(),
-                        range,
-                    });
-                }
+                if id == 0 {}
 
                 match (self.encoding, id) {
                     // Module sections
+                    (Encoding::Module, 0) => {
```

Comment:
I think this needs to match either encoding, right? (both components and core wasm modules)

Reply:
Done.